src/Features/TreeBrowse.php

```diff
@@ -51,104 +51,113 @@
+        $start       = isset($params['start'])
+                            ? $params['start']
                             : 0;
-        $browse      = isset($_REQUEST['browse'])
-                            ? $_REQUEST['browse']
+        $browse      = isset($params['browse'])
+                            ? $params['browse']
                             : 0;
-        $is_deleted  = isset($_REQUEST['is_deleted'])
-                            ? $_REQUEST['is_deleted']
+        $is_deleted  = isset($params['is_deleted'])
+                            ? $params['is_deleted']
                             : 0;
         $criteria    = json_encode($params['criteria']);
 
         $category_list = json_encode(self::getTreeCategoryList($itemtype, $params));
         $no_cat_found  = __("No category found");
 
         $JS = <<<JAVASCRIPT
-        $(function() {
-            var loadingindicator  = $("<div class='loadingindicator'>$loading_txt</div>");
-            $('#items_list').html(loadingindicator); // loadingindicator on doc ready
-            var loadNode = function(cat_id) {
-                $('#items_list').html(loadingindicator);
-                $('#items_list').load('$ajax_url', {
-                    'action': 'getItemslist',
-                    'cat_id': cat_id,
-                    'itemtype': '$itemtype',
-                    'start': $start,
-                    'browse': $browse,
-                    'is_deleted': $is_deleted,
-                    'criteria': $criteria
-                });
-            };
-
-            $('#tree_category').fancytree({
-                // load plugins
-                extensions: ['filter', 'glyph', 'persist'],
-
-                // Scroll node into visible area, when focused by keyboard
-                autoScroll: true,
-
-                // enable font-awesome icons
-                glyph: {
-                    preset: "awesome5",
-                    map: {}
-                },
-
-                persist: {
-                    cookiePrefix: '$itemtype',
-                    expandLazy: true,
-                    overrideSource: true,
-                    store: "auto"
-                },
-
-                // load json data
-                source: {$category_list},
-
-                // filter plugin options
-                filter: {
-                    mode: "hide", // remove unmatched nodes
-                    autoExpand: true, // if results found in children, auto-expand parent
-                    nodata: '{$no_cat_found}', // message when no data found
-                },
-
-                // events
-                activate: function(event, data) {
-                    var node = data.node;
-                    var key  = node.key;
-
-                    loadNode(key);
-                },
-
+        var loadingindicator  = $("<div class='loadingindicator'>$loading_txt</div>");
+        $('#items_list').html(loadingindicator);
+        $.fn.loadNode = function(cat_id) {
```

Comment:
This `loadNode` function is called only at one place (line 136). Maybe you can move its content directly there, instead of storing it in jQuery functions.